library/packages/src/modules/PackageSystem.rb

```diff
@@ -347,12 +348,15 @@ def Installed(package)
       # Unfortunately, initializing Pkg reads the RPM database...
       # so we must avoid it.
       # added --whatprovides due to bug #76181
-      rpm_command = "/usr/bin/rpm -q --whatprovides #{package.shellescape}"
-      output = SCR.Execute(path(".target.bash_output"), rpm_command)
-      log.info "Query installed package with '#{rpm_command}' and result #{output.inspect}"
+      # Use Yast::Execute to prevent false positives (boo#1137992)
+      rpm_command = ["/usr/bin/rpm", "-q", "--whatprovides", package]
+      # We are not raising exceptions in case of return codes different than
+      # 0 or 1. So do not plan to modify the current behavior.
+      output, return_code = Yast::Execute.stdout.on_target!(rpm_command, allow_exitstatus: 0..1)
```

Comment:
@teclator Luckily I was reviewing a Cheetah PR yesterday so I know that the correct name of the option is `allowed_exitstatus`

Reply:
Yep :(, It was already fixed in https://github.com/yast/yast-yast2/pull/944